docs/api/events.md

```diff
@@ -0,0 +1,520 @@
+  - [link](https://tools.ietf.org/html/rfc5545#section-3.8.7.1)
+  - type: **date-time**
+  - e.g. 2020-03-19T14:00:00Z
+
+* `dtstart`: starting date of the `Event`
+  - [link](https://tools.ietf.org/html/rfc5545#section-3.8.2.4)
+  - type: **date-time**
+  - e.g. 2020-03-19T14:00:00Z
+
+* `dtend`: ending date of the `Event`
+  - [link](https://tools.ietf.org/html/rfc5545#section-3.8.2.2)
+  - the `endDate` should be after `startDate`
+  - type: **date-time**
+  - e.g. 2020-03-19T16:30:00Z
+
+* `categories`: categories of the `Event`
+  - [link](https://tools.ietf.org/html/rfc5545#section-3.8.1.2)
+  - type: **text**
+  - e.g. "Exam", "Lecture"
+
+* `rrule`: recurrency rule of the `Event`
+  - [link](https://tools.ietf.org/html/rfc5545#section-3.8.5.3)
+  - type: [**recur**](https://tools.ietf.org/html/rfc5545#section-3.3.10)
+
+## Link Relations
+An event representation:
+* *must* include a link to its context, using the `self` link relation
+* *must* include a link to its creation context, using the `about` link relation
+  - e.g. if an `Event` is created for a specific `Class` then the `about` link relation would refer to that same `Class`
+
+## Example representation
+MIME type: __application/vdn.siren+json__
+```json
+{
+  "class": [ "event" ],
+  "properties": {
+    "type": "event",
+    "properties": {
+      "uid": {
+        "parameters": {},
+        "type": "integer",
+        "value": 1234
+      },
+      "summary": {
+        "parameters": {},
+        "type": "text",
+        "value": "WAD 1st Exam"
+      },
+      "description": {
+        "parameters": {},
+        "type": "text",
+        "value": "First exam of the WAD course during the semester 1920v"
+      },
+      "categories": {
+        "parameters": {},
+        "type": "text",
+        "value": "Exam",
+      },
+      "created": {
+        "parameters": {},
+        "type": "date-time",
+        "value": "2020-02-10T10:34:20Z"
+      },
+      "dtstamp": {
+        "parameters": {},
+        "type": "date-time",
+        "value": "2020-02-10T10:34:20Z"
+      },
+      "dtstart": {
+        "parameters": {},
+        "type": "date-time",
+        "value": "2020-03-19T14:00:00Z"
+      },
+      "dtend": {
+        "parameters": {},
+        "type": "date-time",
+        "value": "2020-03-19T16:30:00Z"
+      },
+    },
+    "subComponents": []
+  },
+  "entities": [
+    {
+      "class": [ "class" ],
+      "rel": [ "/rel/class" ],
+      "properties": {
+        "classId": "wad-s1920v",
+        "termId": "s1920v"
+      },
+      "links": [
+        { "rel": [ "self" ], "href": "/v0/courses/wad/classes/s1920v" },
+        { "rel": [ "term" ], "href": "/v0/terms/s1920v" },
+        { "rel": [ "course"], "href": "/v0/courses/wad" }
+      ]
+    }
+  ],
+  "links": [
+    { "rel": [ "self" ], "href": "/v0/courses/wad/classes/s1920v/events/1234" },
+    { "rel": [ "about" ], "href": "/v0/courses/wad/classes/s1920v" }
+  ]
+}
+```
+
+# `Schedule`
```

Comment:
Should we use Schedule instead of Recurrence Rule? Either one is fine for me, but using both simultaneously would be very inconsistent.